FW/Presentations/Lab_Guides/_L_C_01_Build_Setup_Download_EDK_II_Linux_LabGuide.md

```diff
@@ -0,0 +1,328 @@
+
+```bash
+$ sudo apt install build-essential uuid-dev iasl git python-is-python3
+```
+
+- build-essential - Informational list of build-essential packages
+- uuid-dev - Universally Unique ID library (headers and static libraries)
+- iasl - Intel ASL compiler/decompiler (also provided by acpica-tools)
+- git - support for git revision control system
+- nasm - General-purpose x86 assembler (see below to install 2.15.05 or above)
+- python-is-python3 - Ubuntu 20.04 python command is \'python3\' but edk2 tools use \'python\'
+
+Now need Nasm 2.15.x [Link](https://www.linuxfromscratch.org/blfs/view/cvs/general/nasm.html)
+
+The following is needed for Mounting a .VHD file to use with Simics
+
+```bash
+$ sudo apt install libguestfs-tools
+```
+
+The following will install the QEMU for Intel x86 & 64 bit
+
+```bash
+$ sudo apt install qemu-system-x86-64
+```
+
+**See Lab guide for Ubuntu 16.04 pre-requisites (listed below)**
+
+### **Pre-requisites Ubuntu 16.04**
+
+Instructions from: tianocore wiki Ubuntu_1610 
+- Example Ubuntu 16.04
+
+The following need to be accessible for building Edk2, From the terminal prompt (Cnt-Alt-T):
+```bash
+$ sudo apt-get install build-essential uuid-dev iasl git gcc-5 nasm  python3-distutils
+```
+
+- build-essential - Informational list of build-essential packages
+- uuid-dev - Universally Unique ID library (headers and static libraries)
+- iasl - Intel ASL compiler/decompiler (also provided by acpica-tools)
+- git - support for git revision control system
+- gcc-5 - GNU C compiler (v5.4.0 as of Ubuntu 16.04 LTS)
+- nasm - General-purpose x86 assembler 
+- python3 - distutils - distutils module from the Python standard library
+
+```bash
+$ sudo apt-get install qemu
+```
+
+- Qemu - Emulation with Intel architecture with UEFI Shell 
+
+
+---
+## Slide 5  Pre-requisites Clear Linux* Project 
+
+Example Using Clear Linux* Project 
+
+The following need to be accessible for building Edk2, From the terminal prompt (Ctrl-Alt-T):
+```bash
+$ sudo swupd bundle-add devpkg-util-linux
+```
+
+Devpkg-util-linux - includes bundles for developer tools for writing  "C" Applications included: gcc, nasm, uuid, etc.
+
+```bash
+$ sudo swupd bundle-add kvm-host
+```
+
+Qemu - Emulation with Intel architecture with UEFI Shell 
+
+---
+## Slide 6 Setup Simics Environment
+
+Download and Install the Simics Simulator (both Package Manager & Simics-6 Packages) [Simics® Simulator Public Release](https://www.intel.com/content/www/us/en/developer/articles/tool/simics-simulator.html)
+
+Setup the Simics Simulator
+
+[Simics® Simulator Installation and Get Started](https://www.intel.com/content/www/us/en/developer/articles/guide/simics-simulator-installation.html)
+
+Open the "Intel Simics Package Manager"
+
+1. Using the Start menu in Windows, Left Click on "Windows Key" Lower Left
+2. Scroll down from the scroll bar on the right until "Intel Simics Package Manager"
+
```

Comment:
Replace 1. and 2. with:
1. On Linux, run the ispm-gui application from inside the unpacked directory.
2. Note If the application fails to start on older Linux versions, you may have to start the application with the --no-sandbox option, as ispm-gui --no-sandbox.